renderer/components/image-drop.js

```diff
@@ -0,0 +1,57 @@
+//  validate file format with a list of format supports imgur
+let validateFile = (file) => {
+  let imageFormat = ['jpeg', 'png', 'gif', 'peg', 'apng', 'tiff', 'pdf', 'xcf']
+  return imageFormat.filter((kind) => {
+    let res = new RegExp("\\b(" + kind + ")\\b")
+    return res.test(file.type)
+  })
+}
+
+//  send file via XMLHTTP
+let sendData = (_this, file) => {
+  let fd = new FormData()
+  fd.append('image', file)
+  let xhttp = new XMLHttpRequest()
```

Comment:
Use const, only use let if you realize you need to change this variable later (tip: never do it, use immutable code)

Use Fetch API not XMLHttpRequest